test/setup.js

```diff
@@ -0,0 +1 @@
+require('jsdom-global')()
```

Comment:
What does this do?

Reply:
This loads https://github.com/jsdom/jsdom so that you can simulate a fairly standards-compliant DOM in your tests without needing to run a browser. There's a note [in the Vue documentation](https://vue-test-utils.vuejs.org/guides/choosing-a-test-runner.html) that something that can behave like this is required for the Vue Test Utils to work. Without the test utils, we don't get the `mount` and `shallowMount` methods for pulling individual Vue components into a test.

We're not actually using this yet, though, so I'm fine with leaving this line out until our first test that requires it

Reply:
Leaving a reference to this example test generator from Vue so that I can easily find it later: https://github.com/vuejs/vue-cli/blob/bd7ad93cae9711ca30b05c31c185e1ce56bcb1de/packages/%40vue/cli-plugin-unit-mocha/setup.js#L1